rules/set-fsgroupchangepolicy-value/raw.rego

```diff
@@ -13,7 +13,7 @@ deny[msga] {
     # check securityContext has fsGroupChangePolicy set
     not fsGroupChangePolicySetProperly(pod.spec.securityContext)
 
-    path := "spec.securityContext.fsGroupChangePolicy"
+    path := "spec.securityContext"
```

Comment:
here we can change it to fixpath because we have a value that we recommend to configure (we can choose one of the two that we accept as appropriate)